Enferno.Services.StormConnect.Contracts/Common/Image.cs

```diff
@@ -8,13 +9,26 @@ public class Image
     {
         [DataMember(Order = 1)]
         public string FileTypeName { get; set; }
+
         [DataMember(Order = 2)]
         public string ExternalCode { get; set; }
+
         [DataMember(Order = 3)]
         public string Filename { get; set; }
+
         [DataMember(Order = 4)]
         public string Url { get; set; }
+
         [DataMember(Order = 5)]
         public DateTime? LastModified { get; set; }
+
+        [DataMember(Order = 6)]
+        public List<ImageCulture> Cultures { get; set; }
+
+        [DataMember(Order = 7)]
+        public bool? IsVariantUnique { get; set; }
+
+        [DataMember(Order = 8)]
+        public bool? IsActive { get; set; }
```

Comment:
I took a deeper look at ProductImageProcessor.EnsureFiles and find that there is a lot of meddling with unusedImages that keeps track of the imported images and inactivates those not found in the import but only if Product/VariantField.AdditionalImagesCleanup is set. Otherwise they dont touch any existing image not in import. 
We match existing files on product by ExternalCode or Url if code does not exist (ProductImageProcessor.EnsureDbProductFile). So adding isActive would make it possible to inactivate a specific image by sending an image with ExternalCode set and IsActive=false. But this doesn't rhyme well with AdditionalImagesCleanup. Is it better to keep AdditionalImagesCleanup and remove IsActive from Admin export and Connect import and let Admin import be able to set these flags when importing images. By setting them to true you will only import those images present in the file imported. If set to false, you will add all new images in import but not replace the old ones. This will result in less changes in Connect and still have the same control. We get two clear scenarios in import as well: Add images or replace images. Thoughts on this would be appreciated.